scripts/cem-plugin-angular-wrapper.js

```diff
@@ -35,10 +35,18 @@ ${events.length ? `\n${events.map(event => event.code).join('\n')}\n` : ''}}
 `;
 };
 
-const generateIndex = async (components, outDir) => {
-  const indexSrc = components.map(component => `export * from './${dasherize(component.name)}.component';`).join('\n');
+const generatePublicApis = async (packages, outDir) => {
+  for (const [packageName, files] of packages) {
+    await writeFile(join(outDir, packageName, 'public-api.ts'), files.map(f => `export * from './${f}';`).join('\n').concat('\n'), 'utf8');
 
-  await writeFile(join(outDir, 'index.ts'), indexSrc, 'utf8');
+    await writeFile(join(outDir, packageName, 'ng-package.json'), `{
+  "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
```

Comment:
small thing maybe. but as of `ng-packagr` `17.3.0`, you should use schema `ng-packagr/ng-entrypoint.schema.json` for secondary entrypoints. 
But then again.. if this works..